u4a-component/charts/cluster-component/values.yaml

```diff
@@ -6,7 +6,7 @@
 ingress-nginx:
   tcp:
     # postgresql service
-    5432: "{{ .Release.Namespace }}/postgresql:5432"
+    5432: "baas-system/bc-explorer-postgresql:5432"
```

Comment:
why hardcoded namespace ?

Reply:
This is because u4a components are deployed under the u4a-system namespace, but browsers are deployed under baas-system. There is no parameter to get to baas-system, so it is directly hard-coded here.

Reply:
> This is because u4a components are deployed under the u4a-system namespace, but browsers are deployed under baas-system. There is no parameter to get to baas-system, so it is directly hard-coded here.

i see...

then  we need to let beginers know why we expose this tcp service